### ISSUE TITLE
Input parameters are not removed from XML when morphing Service Task into Receive Task

### ISSUE THREAD

**Issue by barmac** [zeebe-io/zeebe-modeler#248]
__Describe the Bug__

Although input parameters are no longer accessible in the properties panel when Service Task is morphed into Receive Task, they are still present in the XML.

__Steps to Reproduce__

1. Create a Service Task
2. Add input parameters

Task's XML at this point looks like this:

```xml
    <bpmn:serviceTask id="Activity_07bz79w">
      <bpmn:extensionElements>
        <zeebe:ioMapping>
          <zeebe:input source="= source" target="target" />
        </zeebe:ioMapping>
      </bpmn:extensionElements>
    </bpmn:serviceTask>
```

3. Morph into Receive Task
4. Check XML:

```xml
    <bpmn:receiveTask id="Activity_07bz79w">
      <bpmn:extensionElements>
        <zeebe:ioMapping>
          <zeebe:input source="= source" target="target" />
        </zeebe:ioMapping>
      </bpmn:extensionElements>
    </bpmn:receiveTask>
```

Extension elements should be removed so it should look like this:

```xml
    <bpmn:receiveTask id="Activity_07bz79w" />
```

__Expected Behavior__

Input parameters should be removed. User can still recover them via undo.

__Environment__

- OS: MacOS 10.15.6
- Zeebe Modeler Version: 0.10.0-rc.0

**Comment by MaxTru** [comment]
The same issue applies for `calledElement` when morphing from a callActivity to something else.

As @barmac explained to me, we need to fix this in zeebe-bpmn-moddle, compare https://github.com/camunda/camunda-bpmn-moddle/blob/master/lib/extension.js#L27

**Comment by azeghers** [comment]
What we need:
  * [ ] Tests for copy behavior
  * [ ] ZeebeModdleExtension that implements a canCopyProperties
  * [ ] Integrate it in the Zeebe Modeler

**Comment by MaxTru** [comment]
Closed by #274
